src/plugins/data/public/ui/query_editor/editors/default_editor/index.tsx

```diff
@@ -34,6 +34,105 @@ export const DefaultInput: React.FC<DefaultInputProps> = ({
   provideCompletionItems,
   queryStatus,
 }) => {
+  // Create a wrapper for editorDidMount to add our custom status bar
+  const handleEditorDidMount = (editor: monaco.editor.IStandaloneCodeEditor) => {
+    // Call the original editorDidMount function
+    editorDidMount(editor);
+
+    // Add a custom status bar with the "Tab to insert, ESC to close window" message
+    const addCustomStatusBar = () => {
```

Comment:
this seems like duplicate code from the shared.tsx or is it different. if not would it be easier to define this in the query_editor.tsx and pass invoked the method within the editorDidMount function?

Reply:
does this logic actually make more sense (and potentially easier) if it was actually place in the wrapper around the monaco editor that we have

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/opensearch_dashboards_react/public/code_editor/README.md which says it has easier access to the suggestion widget